tests/Browser/Pages/Backend/Books/AdminListBooksTest.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace Tests\Browser\Pages\Backend\Books;
+
+use App\Model\Book;
+use App\Model\User;
+use App\Model\Category;
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Faker\Factory as Faker;
+
+class AdminListBooksTest extends DuskTestCase
+{
+    use DatabaseMigrations;
+    
+    /**
+     * Test view Admin List Books if database has record or empty.
+     *
+     * @return void
+     */
+    public function testListBooks()
+    {
+        $this->makeUserLogin();
```

Comment:
Bỏ cái `makeUserLogin` vào 1 cái hàm gọi là setup nha, hàm này sẽ chạy mỗi khi mình chạy unittest.
Tham khảo: https://stackoverflow.com/questions/33754043/setting-up-phpunit-tests-in-laravel